docs/docs/administer/upgrade.md

```diff
@@ -25,6 +25,12 @@ To upgrade Firezone, follow these steps:
 Occasionally problems arise. If you hit any, please let us know by [filing an
 issue](https://github.com/firezone/firezone/issues/new/choose).
 
+## Upgrading from >= 0.3.11 to >= 0.3.17
```

Comment:
* Did you mean `Upgrading from 0.3.x to >= 0.3.16` as `>= 0.3.11 to >= 0.3.17` is somewhat ambiguous?
* Still need to test this to be sure this is the case
* I think 0.3.16 introduced this

We also need to be very clear here (make it bold, warning label) that any OIDC users added pre-`0.3.16` will need to revoke their tokens for this functionality to work, or the OAuth app needs to be deleted and recreated in their provider's portal.

Reply:
oops this is supposed to be WIP in the OIDC docs branch. Removed it here.